app/src/main/java/com/omkarmoghe/pokemap/views/MainActivity.java

```diff
@@ -43,6 +45,20 @@
     private SharedPreferences sharedPref;
     private int themeId;
 
+    private Snackbar _pokeSnackbar;
```

Comment:
Just so you know the Android Coding Style and Guide lines use m before the name of private fields. In this case it would be mPokeSnackbar.

It looks good to me.